mace/data/atomic_data.py

```diff
@@ -116,11 +123,15 @@ def from_config(
         z_table: AtomicNumberTable,
         cutoff: float,
         heads: Optional[list] = None,
+        **kwargs,  # pylint: disable=unused-argument
```

Comment:
It is so that inherited classes can alutomatically be used with the `HDF5Dataset`, `hdf5_dataset.py`, line 74. 

Let me know if there is a better way to do this.